src/data/set/basic.lean

```diff
@@ -2230,6 +2230,9 @@ set.subset.antisymm (snd_image_prod_subset _ _)
 lemma prod_diff_prod : s.prod t \ s₁.prod t₁ = s.prod (t \ t₁) ∪ (s \ s₁).prod t :=
 by { ext x, by_cases h₁ : x.1 ∈ s₁; by_cases h₂ : x.2 ∈ t₁; simp * }
 
+lemma prod.map_preimage_set_prod (f : α → γ) (g : β → δ) (c : set γ) (d : set δ) :
```

Comment:
```suggestion
lemma _root_.prod.map_preimage_set_prod (f : α → γ) (g : β → δ) (c : set γ) (d : set δ) :
```
or
```suggestion
lemma prod_preimage_prod_map (f : α → γ) (g : β → δ) (c : set γ) (d : set δ) :
```
since you're in namespace `set` here